src/uplift/bot/uplift_bot/bugzilla.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from libmozdata.bugzilla import Bugzilla
+
+
+def cancel_uplift_request(bugzilla_id, branch):
+    '''
+    Cancel an invalid uplift request.
+    '''
+    flag_name = 'approval-mozilla-' + branch
+    flag_reset_payload = {
+        'flags': [
+            {
+                'name': flag_name,
+                'status': 'X'
+            },
+        ],
+    }
+
+    def _attachmenthandler(attachments, bugid, needinfos):
+        for attachment in attachments:
+            flags = [f for f in attachment['flags'] if f['name'] == flag_name]
+            if len(flags) > 0:
+                # Remember who requested this uplift, in order to needinfo them
+                # later.
+                needinfos.add(flags[0]['setter'])
```

Comment:
This part should be in the backend, adding more infos to the bug api output

Reply:
Which "backend" do you mean? (I thought that the backend part of uplift bot was to be refactored/removed soon.)